avocado/plugins/jobs.py

```diff
@@ -56,14 +56,25 @@ def _print_job_tests(tests):
         for test in tests:
             status = test.get("status")
             decorator = output.TEST_STATUS_DECORATOR_MAPPING.get(status)
-            # Retrieve "end" for backward compatibility
-            end = datetime.fromtimestamp(test.get("actual_end", test.get("end")))
+
+            end_timestamp = test.get("actual_end", test.get("end"))
+            if end_timestamp is not None:
+                end = datetime.fromtimestamp(end_timestamp).strftime(date_fmt)
+            else:
+                end = "N/A"
+
+            time_taken = test.get("time")
+            if time_taken is not None:
+                time_str = f"{float(time_taken):5f}"
+            else:
+                time_str = "N/A"
+
             test_matrix.append(
                 (
-                    test.get("id"),
-                    end.strftime(date_fmt),
-                    f"{float(test.get('time')):5f}",
-                    decorator(status, ""),
+                    test.get("id", "N/A"),
+                    end,
+                    time_str,
+                    decorator(status, "") if decorator else "N/A",
```

Comment:
IMO, when there is and unknown status then we should raise an error, because the data are corrupted. Also, IIUIC the `status` is already used in line 58, so this would make avocado crash anyway.